packages/rum/src/rum.ts

```diff
@@ -223,6 +223,7 @@ export function startRum(
       return {
         application_id: applicationId,
         session_id: viewContext.sessionId,
+        user_action: getUserActionReference(),
```

Comment:
On my PR (😆), I missed the `InternalContext` type update, what about enforcing it with:
```ts
getInternalContext: monitor((): InternalContext => {
```